blob/fileblob/fileblob.go

```diff
@@ -565,6 +582,20 @@ func (b *bucket) Delete(ctx context.Context, key string) error {
 }
 
 func (b *bucket) SignedURL(ctx context.Context, key string, opts *driver.SignedURLOptions) (string, error) {
-	// TODO(Issue #546): Implemented SignedURL for fileblob.
+	path, err := b.path(key)
+	if err != nil {
+		return "", err
+	}
+
+	return b.urlSigner.SignedURL(ctx, key, path, opts)
+}
+
+type URLSigner interface {
```

Comment:
Since this is introducing new exported API, it should have a doc comment explaining what the interface is. Usually for one-method interfaces like this, the method documentation goes in the interface doc comment, like in [`io.Reader`](https://godoc.org/io#Reader).

Reply:
PTAL